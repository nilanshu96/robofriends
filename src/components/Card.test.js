import { render } from "@testing-library/react";
import Card from "./Card";

test("loads a single card", () => {
    const { getAllByTestId } = render(
        <Card id="1" name="Mr 1" email="baroqueworks@gmail.com" />
    );
    const cards = getAllByTestId("robocard");
    expect(cards.length).toEqual(1);
});

test("renders correctly", () => {
    const { container } = render(
        <Card id="1" name="Mr 1" email="baroqueworks@gmail.com" />
    );
    expect(container.innerHTML).toMatchSnapshot();
})