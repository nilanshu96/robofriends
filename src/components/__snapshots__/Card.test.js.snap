// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `"<div data-testid=\\"robocard\\" class=\\"tc bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5\\"><img src=\\"https://robohash.org/1.png?size=200x200\\" alt=\\"robot\\"><div><h2>Mr 1</h2><p>baroqueworks@gmail.com</p></div></div>"`;
